app/controllers/clearance/sessions_controller.rb

```diff
@@ -33,4 +40,8 @@ def url_after_create
   def url_after_destroy
     sign_in_url
   end
+
+  def url_for_signed_in_users
+    url_after_create
```

Comment:
no problem. yes, this is for signing in, so in this context `create` just means that the user is signing in.